src/test/java/uk/gov/ida/matchingserviceadapter/resources/MatchingServiceResourceTest.java

```diff
@@ -47,7 +47,7 @@ public void testMatching() throws ParserConfigurationException {
 
```

Comment:
I'm not so sure this tests adds value. We don't tend to test Resources and we should have an acceptance test which checks we can handle matching requests

Reply:
There appears to be existing tests that checks that things are called. I think whoever put this in was trying to follow it to ensure certain things are called. We've added the explicit ```times(1)``` bit to make it a bit more verbose. If you still think that it doesn't really add much value we can remove it.

BTW - I believe there are acceptance tests. If not we'll definitely have them when the full happy path is completed. (At the moment, we still aren't there yet.)

Reply:
I think its best to remove it. Tests which assert on implementation specifics rather than inputs and outputs tend to be brittle and make refactoring harder. They also don't articulate the expected behaviour but rather the implementation detail. We should be testing at the level above that responses are returned from this method.

Reply:
Deleted